tequilapi/endpoints/transactor.go

```diff
@@ -67,6 +68,47 @@ func (te *transactorEndpoint) TransactorFees(resp http.ResponseWriter, _ *http.R
 	utils.WriteAsJSON(fees, resp)
 }
 
+// swagger:operation POST /transactor/topup ErrorMessageDTO
+// ---
+// summary: tops up myst to the given identity
+// description: tops up myst to the given identity
+// parameters:
+// - in: body
+//   name: body
+//   description: top up request body
+//   schema:
+//     $ref: "#/definitions/TopUpRequestDTO"
+// responses:
+//   202:
+//     description: top up request accepted
+//   500:
+//     description: Internal server error
+//     schema:
+//       "$ref": "#/definitions/ErrorMessageDTO"
+//   400:
+//     description: Bad request
+//     schema:
+//       "$ref": "#/definitions/ErrorMessageDTO"
+func (te *transactorEndpoint) TopUp(resp http.ResponseWriter, request *http.Request, _ httprouter.Params) {
+	topUpDTO := &transactor.TopUpRequest{}
+
+	if request.ContentLength > 0 {
```

Comment:
Not sure about this check. If would suggest to remove it and simply allow to decode body to TopUpRequest and let if fail because now I will be able to pass empty content and call transactor.TopUp